shell/platform/windows/window_unittests.cc

```diff
@@ -272,5 +284,78 @@ TEST(MockWindow, Paint) {
   window.InjectWindowMessage(WM_PAINT, 0, 0);
 }
 
+TEST(MockWindow, PointerHitTest) {
```

Comment:
I know we haven't been great about this but consider adding a comment above this and a link to the issue so future you can see what crash it tests for.

Reply:
Sadly this doesn't test the regression, it only adds missing unit tests. AFAICT the only way to test the regression would be to run tests on Windows 7 to trigger dynamic linking.

Reply:
Ah yep -- hadn't yet read the tests when I wrote that; just in general we should add comments above the tests, and where they're testing a regression, a link to the issue.

There's no simple way to test against the crash caused by failute to resolve the function at runtime, but we can test that the code that uses it behaves nicely in response to a mock proc resolver returning false, which is a slightly higher-level check, but still important to verify since that's what we'll be doing on Windows 7. That test should then be tagged with the bug.

Reply:
Sounds good. I added a comment to each test, and, tagged the crash's issue on the `UnknownPointerTypeSkipsDirectManipulation` test